support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -106,17 +106,29 @@ DomainCreator:
         InitialValues: [[0.0, 0.0, 0.0], [0.0, 0.0, 0.0], [0.0, 0.0, 0.0]]
       ShapeMapA:
         LMax: &LMax 10
-        InitialValues: Spherical
+        InitialValues:
+          H5Filename: &HorizonsFile {{ HorizonsFile }}
```

Comment:
Have you tried this? I think this won't be wrapped in quotes, which could be a problem for paths that have spaces, and is definitely a problem in the list of subfile names below.

Reply:
Yeah this worked just fine. However, I didn't think about the spaces. I'll put these in quotes to account for that